pcsx2/Gif_Unit.h

```diff
@@ -258,9 +278,10 @@ struct Gif_Path {
 
 	// If completed a GS packet (with EOP) then returned GS_Packet.done = 1
 	// MTVU: This function only should be called called on EE thread
-	GS_Packet ExecuteGSPacket() {
+	GS_Packet ExecuteGSPacket(bool &done) {
```

Comment:
Pointer is error prone. You need to check if it is null, otherwise someone might use it badly on the future.